tests/app/views/contexts/test_submitted_response_context.py

```diff
@@ -82,6 +83,21 @@ def test_build_view_submitted_response_context_submitted_text_with_trad_as(app:
         )
 
 
+def test_view_submitted_response_expired_is_true_when_submitted_at_greater_than_the_expiration_time(
+    app: Flask,
+):
+    with app.app_context():
+        questionnaire_store = fake_questionnaire_store()
+        questionnaire_store.submitted_at = datetime.now(timezone.utc) - timedelta(
+            minutes=46
```

Comment:
It would be better if the VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS value with something added was used here.